src/software/simulation/er_force_simulator.cpp

```diff
@@ -111,18 +73,110 @@ void ErForceSimulator::setBallState(const BallState& ball_state)
     *(command_simulator->mutable_ssl_control())     = *simulator_control;
     *(simulator_setup_command->mutable_simulator()) = *command_simulator;
 
-
     er_force_sim->handleSimulatorSetupCommand(simulator_setup_command);
 }
 
 void ErForceSimulator::addYellowRobots(const std::vector<RobotStateWithId>& robots)
 {
-    // TODO (#2283): add robots
+    addRobots(robots, gameController::Team::YELLOW);
 }
 
 void ErForceSimulator::addBlueRobots(const std::vector<RobotStateWithId>& robots)
 {
-    // TODO (#2283): add robots
+    addRobots(robots, gameController::Team::BLUE);
+}
+
+void ErForceSimulator::addRobots(const std::vector<RobotStateWithId>& robots,
+                                 gameController::Team side)
+{
+    auto simulator_setup_command = std::make_unique<amun::Command>();
+    simulator_setup_command->mutable_simulator()->set_enable(true);
+
+    robot::Specs ERForce;
+    robotSetDefault(&ERForce);
+
+    // Initialize Team Robots at the bottom of the field
+    if (side == gameController::Team::BLUE)
+    {
+        auto team = simulator_setup_command->mutable_set_team_blue();
+        for (unsigned int i = 0; i < robots.size(); i++)
+        {
+            auto* robot = team->add_robot();
+            robot->CopyFrom(ERForce);
+            robot->set_id(i);
+        }
```

Comment:
Can we use a for each loop instead and use the robot id? Same with all the other for loops
```suggestion
        for (const auto & robot_state_with_id: robots)
        {
            auto* robot = team->add_robot();
            robot->CopyFrom(ERForce);
            robot->set_id(robot_state_with_id.id);
        }
```